userguide/content/en/docs/adding-content/shortcodes/index.md

```diff
@@ -706,4 +706,50 @@ This text does not appear in the output if `buildCondition = "bar" is set in you
 
 If you are using this shortcode, note that when evaluating the conditions, substring matches are matches as well. That means, if you set `include-if="foobar"`, and `buildcondition = "foo"`, you have a match!
 
+## Accordion
+
+Sometimes it's very useful to have an accordion element at hand when authoring content. One common use-case is FAQ section where only one section (= accordion item) will be open at any time. The following sample showcases a accordion presenting the four principles of object-oriented programming: 
+
+### Sample accordion: Object-oriented programming
```

Comment:
```suggestion
### Sample accordion: object-oriented programming
```
changed `Object-oriented` to `object-oriented` to match case in previous section

Reply:
Changed as requested.